pkg/ui/src/components/core/typography.styl

```diff
@@ -53,6 +53,8 @@ $spacing-large = 32px
 $spacing-x-large = 40px
 $spacing-xx-large = 48px
 
+$max-window-width = 1350px
```

Comment:
This does not belong in `typography.styl`. Can we move to `layout-vars.styl` and also use it wherever we have 1350px mentioned?